.github/dependabot.yaml

```diff
@@ -6,9 +6,6 @@ updates:
     labels: ["area/ci", "dependencies"]
     schedule:
       interval: "weekly"
-    # Project maintainers and the Wild Watermelon team
-    reviewers:
-      - "weaveworks/wild-watermelon"
     # Only do security updates not version updates.
     open-pull-requests-limit: 0
```

Comment:
Why is this deactivated for version bumps?

Reply:
Thanks for the review @chanwit!

Any inputs on this comment? Should this be enabled in another PR?